deleter.py

```diff
@@ -344,6 +344,35 @@ def delete_parts(self, ids_to_remove):
             url = D42_URL + f
             r = requests.delete(url, headers=self.headers, verify=False)
             i += 1
+			
+    def delete_serviceinstances(self, ids_to_remove):
+        """
+		Added - 03/24/2020 - Not part of Wipe all
```

Comment:
1) Did not want to add via the Wipe All because not sure what data integrity issues might arise. Folks have not used it in the past so I decided to leave off, since I did not have a chance to test/validate.
2) No sure why wipe all does not include IPs. Was that way when I updated the script.